src/components/Message.vue

```diff
@@ -237,7 +237,7 @@ export default {
 				envelope: this.envelope,
 				message: this.message,
 			})
-			this.$store.dispatch('deleteMessage', this.message)
+			this.$store.dispatch('deleteMessage', this.message.accountId, this.message.folderId, this.message.id)
```

Comment:
I like it especially in this file. We currently fetch the `message` object *and* the `envelope` object. While we need the latter just for passing it to the actions. If more actions work with key-only, we can remove this overhead :rocket: